src/helpers/DiffHelpers.ts

```diff
@@ -0,0 +1,77 @@
+import {
+  Operation,
+  ReplaceOperation,
+  AddOperation,
+  getValueByPointer,
+} from 'fast-json-patch';
+
+export interface DiffOutput {
+  action: string;
+  path: string;
+  index?: number;
+  before?: any;
+  after?: any;
+}
+
+type ValueOperation = ReplaceOperation<any> | AddOperation<any>;
```

Comment:
Should these types and interfaces be in a different file like `types.ts`?

I'm thinking of moving these types as well as some types we make in the future should be moved inside a `types.ts` file. What do you think?

Reply:
It would be great :) For example, we have `types.ts` in the react-component repo.